meta/pleroma/emoji-index/QueerCats/index.json

```diff
@@ -0,0 +1,10 @@
+{
+    "QueerCats": {
+      "src_sha256": "fab0006c409ce93354a5961c015629d2b79f12da5ef8da553e623a593157f0f6",
+      "src": "https://github.com/ZoeBijl/QueerCats/releases/download/v0.2/Queer.Cats.v0.2.zip",
+      "license": "CC0",
+      "homepage": "https://github.com/ZoeBijl/QueerCats",
+      "files": "https://raw.githubusercontent.com/ZoeBijl/QueerCats/master/meta/pleroma/emoji-index/QueerCats/QueerCats.json",

```

Comment:
Is there a reason we can’t use https://zoebijl.github.io/QueerCats/meta/pleroma/emoji-index/QueerCats/QueerCats.json?

Reply:
Nope... New commit pushed to address this.

Reply:
Woohoo! Thank you very much. If possible I’d like to rely on as few services as possible.